### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by darwinaraqueo**
## Summary
- This feature allow send a message to user with an expiring rent

## Screenshot
image

## Trello card
- https://trello.com/c/IqtxNqEy/41-notificar-al-usuario-cuando-un-rent-se-vence

### CODE REVIEW

File: app/jobs/rent_expiration_job.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class RentExpirationJob
+  include Sidekiq::Worker
+  def perform
+    rents = Rent.where('end_date = ?', Time.zone.now)
```
Comment (santyalcala) [inline app/jobs/rent_expiration_job.rb]: This should be `Date.today` as we talked, `Time.zone.now` returns a Datetime and `end_date` is a `Date`

File: config/sidekiq.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+:schedule:
+  hello_world:
+    cron: '0 0 9 * * * America/Sao_Paulo'
```
Comment (santyalcala) [inline config/sidekiq.yml]: The Cron code is not okay, its max 5 items and not 6.

Comment (darwinaraqueo) [reply config/sidekiq.yml]: plis look this https://github.com/Moove-it/sidekiq-scheduler#time-zones

File: app/jobs/rent_expiration_job.rb (hunk 1 above)
Comment (darwinaraqueo) [reply app/jobs/rent_expiration_job.rb]: well rubocop says 'Do not use Date.today without zone. Use Time.zone.today instead'.

File: config/sidekiq.yml (hunk 2 above)
Comment (santyalcala) [reply config/sidekiq.yml]: 👍

File: app/jobs/rent_expiration_job.rb (hunk 1 above)
Comment (santyalcala) [reply app/jobs/rent_expiration_job.rb]: Its okay then.

**Pull request merged**